tests/cross/src/test/scala/tests/pc/CompletionScaladocSuite.scala

```diff
@@ -197,4 +199,88 @@ object CompletionScaladocSuite extends BaseCompletionSuite {
        |}
        |""".stripMargin
   )
+
+  checkEdit(
+    "scaladoc-defdef-returns-unit",
```

Comment:
since this is the `CompletionScaladocSuite` I think you can avoid the `scaladoc-` prefix in all tests